docs/concepts/13-security.md

```diff
@@ -26,14 +26,16 @@ All the audits of Lockup contracts can be found [here](https://github.com/sablie
 
 ## Bug Bounty
 
-The Sablier Protocol is subject to a bug bounty program per the terms outlined in `Security.md` of each protocol.
+The Sablier Protocol is subject to a bug bounty program per the terms outlined in the `SECURITY.md` files:
```

Comment:
Since the protocol assumptions are mentioned in the bug bounty page, we can remove it from here. Wdyt? Reading `SECURITY.md` is followed by the the bug bounty program page so once a user has gone through the notion page, this would become reudundant for him.

Reply:
Yeah that's a good point, let's remove

Reply:
https://github.com/sablier-labs/docs/pull/302/commits/9682f66f27535066211187ede41e0e02f7d32ccf